store.go

```diff
@@ -51,10 +51,11 @@ type Store struct {
 	mu   sync.Mutex
 	path string
 
-	id          uint64 // unique node id
-	clusterID   atomic.Value
-	dbs         map[string]*DB
-	subscribers map[*Subscriber]struct{}
+	id               uint64 // unique node id
+	clusterID        atomic.Value
+	dbs              map[string]*DB
+	subscribers      map[*Subscriber]struct{}
+	primaryTimestamp atomic.Int64 // ms since epoch of last update from primary. -1 if primary
```

Comment:
Does it make more sense to have it be `0` if it's the primary since there's no lag? The `-1` could be reserved for if the node is not connected to the primary (or hasn't received a heartbeat).